src/odemis/util/conversion.py

```diff
@@ -104,9 +104,12 @@ def rgb_to_frgb(rgb):
     :return: (float, float, float)
 
     """
-
+    # check if the tuple consists of three values
     if len(rgb) != 3:
-        raise ValueError("Illegal RGB colour %s" % rgb)
+        raise ValueError("Illegal RGB colour %s" % (rgb,))
+    # check if the tuple values are valid RGB values
+    if not numpy.all(tuple(0 <= c <= 255 for c in rgb)):
```

Comment:
You don't need to use `numpy.all()`, you can just use `all()`. You also don't need to convert to a tuple then.
```suggestion
    if not all(0 <= c <= 255 for c in rgb):
```